runtime/gc_glue_java/CompactDelegate.cpp

```diff
@@ -143,6 +147,8 @@ MM_CompactDelegate::mainSetupForGC(MM_EnvironmentBase *env)
 		for (uintptr_t i = 0; i < regionExtension->_maxListIndex; i++) {
 			MM_OwnableSynchronizerObjectList *list = &regionExtension->_ownableSynchronizerObjectLists[i];
 			list->startOwnableSynchronizerProcessing();
+			MM_ContinuationObjectList *list2 = &regionExtension->_continuationObjectLists[i];
+			list2->startContinuationProcessing();
```

Comment:
put these 2 just into one line, without introducing list2 local
lines for ownableSynchronizers will hopefully be gone soon.